fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/StaffTest.java

```diff
@@ -183,13 +185,15 @@ public void testStaffUpdate() {
         Assert.assertEquals(mobileNo, changes.get("mobileNo"));
     }
 
+    @SuppressWarnings("JUnit4TestNotRun")
     public void testStaffUpdateLongExternalIdError() {
         final HashMap<String, Object> map = new HashMap<>();
         map.put("externalId", Utils.randomStringGenerator("EXT", 98));
 
         StaffHelper.updateStaff(requestSpec, responseSpecForValidationError, 1, map);
     }
```

Comment:
@vorburger or @awasum this looks like a test any idea why they are missing @Test.

Reply:
@percyashu I suppose someone just "forgot" to add `@Test` ? 

What I suggest you do is do add `@Test` for all these cases (instead of `@SuppressWarnings("JUnit4TestNotRun")`), and try running them. If they fail, slap an `@Ignore` on those that do fail - and then subsequently but in a later separate PR see if you can fix them (or someone else can do that, as part of future work to review all ignored tests). Sounds like a plan?